src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -813,6 +815,18 @@ const AudioPage = () => {
                     <Slider className="w-60" value={windVolume + 50} onInput={(value) => setWindVolume(value - 50)} />
                 </div>
             </div>
+            <div className="py-4 flex flex-row justify-between items-center">
+                <span className="text-lg text-gray-300">Passenger Ambience</span>
+                <div className="flex flex-row items-center py-1.5">
+                    <Toggle value={!!passengerAmbienceEnabled} onToggle={(value) => setPassengerAmbienceEnabled(value ? 1 : 0)} />
```

Comment:
Shouldn't that be `value ? 0 : 1`?

Reply:
No

Reply:
The Toggle component provides you with the new value.

Reply:
Ok